Source/SwiftLintFramework/Reporters/JUnitReporter.swift

```diff
@@ -10,17 +10,17 @@ public struct JUnitReporter: Reporter {
     }
 
     public static func generateReport(_ violations: [StyleViolation]) -> String {
-        return "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n<testsuites><testsuite>" +
+        return "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n<testsuites><testsuite>\n" +
             violations.map({ violation -> String in
-                let fileName = (violation.location.file ?? "<nopath>").escapedForXML()
+                let fileName = (violation.location.relativeFile ?? "<nopath>").escapedForXML()
```

Comment:
can you update the tests so `file != relativePath`?

Reply:
Not sure I understand what you mean.

Reply:
... what test file should i be updating?

Reply:
check `ReporterTests.swift`